frontend/src/styles/global.scss

```diff
@@ -15,7 +15,10 @@ Only 400 (`normal`), 500 (`var(--font-medium)`), 600 (`var(--font-semibold)`), o
 @import 'functions';
 @import 'mixins';
 @import 'utility-legacy';
-@import 'utilities';
+@import 'colors';
+@tailwind base;
+@tailwind components;
```

Comment:
Web Analytics is pretty borked!

<img width="1194" alt="Screenshot 2023-12-06 at 09 33 13" src="https://github.com/PostHog/posthog/assets/2536520/e7c6218c-e6ee-4afa-b168-daa111db07ad">

Reply:
Fixed!